### DISCUSSION TITLE
Control has already been added to another page: {}".format(ctrl)

### DISCUSSION THREAD

**Discussion by hoangde2811** [flet-dev/flet#1198] [Q&A]
Dear developer!
I create a progress to wait for data to load, for that I used dialog and have await page.update_async , when opening a browser tab there is no problem but opening another browser tab gives an error as attached picture.
(https://user-images.githubusercontent.com/87251371/226081457-51a79651-f463-4bcb-bc86-77e86108cdc7.png)
(https://user-images.githubusercontent.com/87251371/226088895-0030186d-c778-4501-b6ca-c57e93c2f873.png) and here is my progress dialog file.
(https://user-images.githubusercontent.com/87251371/226089756-dffb1db1-2df3-40f8-855f-52231edf759b.png) and here is my main file
Please help me so I can run multiple tabs.
Sincerely thank you.

**Comment by FeodorFitsner** [comment]
Can you paste a code reproducing the issue?

**Reply by hoangde2811** [reply]
Dear developer!
I create a progress to wait for data to load, for that I used dialog and have await page.update_async , when opening a browser tab there is no problem but opening another browser tab gives an error as attached picture.
(https://user-images.githubusercontent.com/87251371/226081457-51a79651-f463-4bcb-bc86-77e86108cdc7.png)
(https://user-images.githubusercontent.com/87251371/226088895-0030186d-c778-4501-b6ca-c57e93c2f873.png) and here is my progress dialog file.
(https://user-images.githubusercontent.com/87251371/226089756-dffb1db1-2df3-40f8-855f-52231edf759b.png) and here is my main file
Please help me so I can run multiple tabs.
Sincerely thank you.
